### DISCUSSION TITLE
Viewing correct sunrise and sunset times in other locations

### DISCUSSION THREAD

**Discussion by SJC117** [Stellarium/stellarium#3027] [General]
I am using stellarium-web. When I view the sky in locations in other time zones, the time changes, but the time that the Sun rises and sets continues to align to my time. For example, a location 6 hours east shows that the Sun rises at noon instead of 6:00 am. How can I change it so the Sun is rising and setting at the correct time for the location I have chosen?

**Comment by gzotti** [comment]
Stellarium-web is for casual users and has a much simpler interface (with less options), the desktop version (which is discussed in this forum) is for people with more specific demands.

**Comment by SJC117** [comment]
Where can I find help for Stellarium-web?
